src/wallet/wallet_state_manager.py

```diff
@@ -529,14 +529,18 @@ async def coins_of_interest_added(self, coins: List[Coin], height: uint32) -> Li
 
         if prev is not None:
             # include last block
-            pool_rewards.add(bytes32(prev.height.to_bytes(32, "big")))
-            farmer_rewards.add(std_hash(std_hash(prev.height)))
+            pool_rewards.add(std_hash(bytes32(prev.height.to_bytes(32, "big")) + self.constants.GENESIS_CHALLENGE))
```

Comment:
How about creating `farming_height` which is 128 bits of genesis challenge and 128 bits of index? That way the farming reward coins are still easily visible.

Reply:
```
network_offset = int.from_bytes(self.constants.GENESIS_CHALLENGE[:16], "big") << 128
farmed_height = prev.height + network_offset
```
and then use `farmed_height` where we have `prev.height`.

Reply:
I think it's cleaner to have the whole genesis challenge be used, but i'm fine with that approach too. @bramcohen ?

Reply:
I'm not sure about using only half of the genesis challenge. Also we only need 32 bits for the height.

What about:
1. 96 bits of  H(genesis challenge + height) and 32 bits of height
2. 96 bits of  H(H(genesis challenge + height)) and 32 bits of height